src/event.cpp

```diff
@@ -798,6 +815,9 @@ void Event::setIsMissedCall(bool isMissed)
 {
     d->flags.isMissedCall = isMissed;
     d->propertyChanged(Event::IsMissedCall);
+    if (isMissed) {
+        setIncomingStatus(Event::Missed);
```

Comment:
Infinite loop here. setIncomingStatus() calls setIsMissedCall()

Reply:
Thank you, it will save me some debugging time.